### PR TITLE
Cut lines > 79

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

File: delivery_carrier_label_default_webkit/report/shipping_label_report.py [hunk 1]
```diff
@@ -33,8 +33,10 @@ def __init__(self, cr, uid, name, context):
             'uid': uid,
         })
 
+module_name = 'delivery_carrier_label_default_webkit'
+template = '/report/template/shipping_label.mako'
 report_sxw.report_sxw(
     'report.delivery.shipping_label',
     'stock.picking',
-    'delivery_carrier_label_default_webkit/report/template/shipping_label.mako',
+    module_name + template,
```
Comment (pedrobaeza) [inline delivery_carrier_label_default_webkit/report/shipping_label_report.py:41]: Is this change the same?

Comment (guewen) [inline delivery_carrier_label_default_webkit/report/shipping_label_report.py:41]: why would it be different?

Comment (pedrobaeza) [inline delivery_carrier_label_default_webkit/report/shipping_label_report.py:41]: Sorry, I didn't see the variables above. Of course is the same.

**Comment by gurneyalex** [comment]
LGTM :+1: and I'll probably fast track this once Travis shows some green 

thanks!

**Comment by pedrobaeza** [comment]
:+1:  for me too

**Pull request merged**